### PR TITLE
Residual Policy Learning

### PR DESCRIPTION
**Pull request by AntoineTheb**

### CODE REVIEW

File: article/_posts/2019-06-27-RPL.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+$$\pi_\theta(s) = \pi(s) + f_\theta(s)$$
+
+Because the policy $$\pi$$ is fixed, the gradient of the policy $$\pi_\theta$$ is given by:
+
+$$\nabla_{\theta}\pi_{\theta}(s) = \nabla_{\theta}f_{\theta}(s)$$
+
+Therefore, RPL can be used to improve on non-differentiable policies.
+
+To use these two policies together, let's remind ourselves that the policy $$\pi$$ acts in the MDP $$M = (S,A, R, T, \gamma)$$ where $$S$$ is the set of all possible states, $$A$$ the set of all possible actions, $$R$$ the reward for taking action $$a \in A$$ at $$s \in S$$, $$T(s, a, s')$$ the transition probabilities and $$\gamma$$ the discount factor.
```
Comment (jhlegarreta) [inline article/_posts/2019-06-27-RPL.md]: Provide the meaning of the acronym MDP?

File: article/_posts/2019-06-27-RPL.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+![](/article/images/RPL/fig5.jpeg)
+
+## Initializing the residuals
+
+One thing to make sure is that the residuals don't make the initial policy worse. If the initial policy is _perfect_, the residuals should have no influence. To alleviate this problem, the authors initialize the residual function so that $$f_{\theta}(s) = 0$$ by making the weights of the last layer of their network all zero.
+
+## RPL with Actor-Critic Methods
+
+Actor-Critic method where the gradient of the actor depends on the critic and a bad critic may make the actor _worse_. The authors, therefore, propose to train the critic alone first until its loss dips below a threshold $$\Beta$$, which they fixed at $$\Beta = 1.0$$
```
Comment (jhlegarreta) [inline article/_posts/2019-06-27-RPL.md]: `Actor-Critic method where the gradient of the actor depends on the critic and a bad critic may make the actor _worse_.`
The sentence seems to be missing the verb/unfinished.

Comment (AntoineTheb) [reply article/_posts/2019-06-27-RPL.md]: Woops, fixed in "99802cd"

File: article/_posts/2019-06-27-RPL.md (hunk 1 above)
Comment (AntoineTheb) [reply article/_posts/2019-06-27-RPL.md]: Fixed in 99802cd

**Pull request merged**